doc/source/release-notes/Percona-XtraDB-Cluster-5.6.38-26.23.rst

```diff
@@ -0,0 +1,64 @@
+.. rn:: 5.6.38-26.23
+
+===================================
+Percona XtraDB Cluster 5.6.38-26.23
+===================================
+
+Percona is glad to announce the release of
+Percona XtraDB Cluster 5.6.38-26.23 on January 24, 2018.
+Binaries are available from the `downloads section
+<http://www.percona.com/downloads/Percona-XtraDB-Cluster-56/>`_
+or from our :ref:`software repositories <installation>`.
+
+Percona XtraDB Cluster 5.6.38-26.23 is now the current release,
+based on the following:
+
+* `Percona Server 5.6.38 <https://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.38-83.0.html>`_
+
+* Galera/Codership WSREP API Release 5.6.38
+
+* Galera Replication library 3.21
+
+All Percona software is open-source and free.
+
+
+
+Fixed Bugs
+==========
+
+* :jirabug:`PXC-889`: Fixed starting a node when :variable:`wsrep_provider`
+  cannot load the galera library, to protect against node inconsistency by
+  operating in a standalone mode. Bug fixed :bug:`1728774`
+
```

Comment:
We should reword this, since this is unclear as to what actually changed.

Fixed an issue where a node with an invalid :variable:`wsrep_provider` was allowed to start up and operate in standalone mode, which could lead to data inconsistency.